.circleci/config.yml

```diff
@@ -117,6 +131,14 @@ workflows:
             branches:
               only: master
       - test
+      - publish:
+          requires:
+            - test
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
Should this be master?  
Would it not publish a potential WIP branch once pushed to Github?

Reply:
Oh right I see it's a tag, is this vital?

Reply:
Yep. It's the mechanism for ensuring that it only tries to publish changes. From here https://circleci.com/blog/publishing-npm-packages-using-circleci-2-0/ Otherwise the step will fail if CircleCI tries to publish the same version of a package to NPM

Reply:
(We currently have that problem in the `fb-components-core` package, with lots of false negatives, because we don't always change the package version, and CircleCI fails when it tries to publish)

Reply:
Cool ok I see, need to ensure this is documented somewhere